### PR TITLE
Addtribler

### PR DESCRIPTION
**Pull request by i-am-xhy**
adds exitnode with no working multichain

### CODE REVIEW

**Comment by StefanBoodt** [comment]
Why did you destroy the fix on the addrgen with this PR? This should not be merged this way. Nice that tribler submodule is gone though.

**Comment by StefanBoodt** [comment]
In addition why is addrgen.sh changed. I do not see why this change was made.

File: run.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@ nohup java -jar selenium-server-standalone-2.53.0.jar &
 #################
 #Start the agent#
 #################
-PYTHONPATH=${PYTHONPATH}:. python agent/agentCore.py
+nohup PYTHONPATH=${PYTHONPATH}:. python agent/agentCore.py &
```
Comment (StefanBoodt) [inline run.sh]: What does this change do?

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ Selenium==2.53.1
 paramiko==1.17.0
 https://download.electrum.org/2.6.4/Electrum-2.6.4.tar.gz
 pexpect-u==2.5.1
+Yappi
```
Comment (StefanBoodt) [inline requirements.txt]: No version specified.

File: run.sh (hunk 1 above)
Comment (i-am-xhy) [inline run.sh]: it runs it in a unbounded state. so that if the terminal gets closed the process doesnt get closed

**Comment by StefanBoodt** [comment]
I do not think that Exitnode.py can be refactored within 1 year of work for 1 person.

**Comment by StefanBoodt** [comment]
This pull request fixes #32 but I wonder how long #15 will stay fixed.

File: .gitmodules [hunk 3]
```diff
@@ -1,9 +1,6 @@
-[submodule "tribler"]
-	path = tribler
-	url = https://github.com/Tribler/tribler.git
 [submodule "docs"]
 	path = docs
 	url = https://git.overleaf.com/5049466xzvfzv
-[submodule "addrgen-master"]
+[submodule "addrgen2"]
 	path = addrgen
```
Comment (StefanBoodt) [inline .gitmodules]: Why is this altered. It was refactored in another branch, which is now inside the master branch, and including this in the pull request will simply undo that refactoring.

**Pull request merged**